docs/configuration/sendrecvoptions.rst

```diff
@@ -9,37 +9,151 @@ Send & Recv Options
+      - **Be careful**, read the :ref:`note on property replication below <job-note-property-replication>`.
+    * - ``large_blocks``
+      - ``-L``
+      - **Potential data loss on OpenZFS < 2.0**, see :ref:`warning below <job-send-options-large-blocks>`.
+    * - ``compressed``
+      - ``-c``
+      -
+    * - ``embbeded_data``
+      - ``-e``
+      -
+    * - ``saved``
+      - ``-S``
+      -
+
+.. _job-send-options-encrypted:
+
+``encrypted``
+-------------
+
+The ``encrypted`` option controls whether the matched filesystems are sent as `OpenZFS native encryption <http://open-zfs.org/wiki/ZFS-Native_Encryption>`_ raw sends.
+More specifically, if ``encrypted=true``, zrepl
 
 * checks for any of the filesystems matched by ``filesystems`` whether the ZFS ``encryption`` property indicates that the filesystem is actually encrypted with ZFS native encryption and
 * invokes the ``zfs send`` subcommand with the ``-w`` option (raw sends) and
 * expects the receiving side to support OpenZFS native encryption (recv will fail otherwise)
 
 Filesystems matched by ``filesystems`` that are not encrypted are not sent and will cause error log messages.
 
-If ``encryption=false``, zrepl expects that filesystems matching ``filesystems`` are not encrypted or have loaded encryption keys.
+If ``encrypted=false``, zrepl expects that filesystems matching ``filesystems`` are not encrypted or have loaded encryption keys.
+
+.. NOTE::
+
+   Use ``encrypted`` instead of ``raw`` to make your intent clear that zrepl must only replicate filesystems that are actually encrypted by OpenZFS native encryption.
+   It is meant as a safeguard to prevent unintended sends of unencrypted filesystems in raw mode.
+
+
+.. _job-send-options-large-blocks:
+
+``large_blocks``
+----------------
+
+This flag should not be changed after initial replication.
+Prior to `OpenZFS commit 7bcb7f08 <https://github.com/openzfs/zfs/pull/10383/files#diff-4c1e47568f46fb63546e984943b09a3e6b051e2242649523f7835bbdfe2a9110R337-R342>`_
+it was possible to change this setting which resulted in **data loss on the receiver**.
+The commit in question is included in OpenZFS 2.0 by prohibiting receives of incremental streams with a flipped setting.
+
+.. WARNING::
+
+   This bug has **not been fixed in the OpenZFS 0.8 releases** which means that changing this flag after initial replication might cause **data loss** on the receiver.
 
 .. _job-recv-options:
 
 Recv Options
 ~~~~~~~~~~~~
 
-:ref:`Sink<job-sink>` and :ref:`pull<job-pull>` jobs have an optional ``recv`` configuration section.
-However, there are currently no variables to configure there.
+:ref:`Sink<job-sink>` and :ref:`pull<job-pull>` jobs have an optional ``recv`` configuration section:
+
+::
+
+   jobs:
+   - type: pull
+     recv:
+       properties:
+         inherit:
+           - "mountpoint"
+         override: {
+           "org.openzfs.systemd:ignore": "on"
+         }
+     ...
+
+.. _job-recv-options--inherit-and-override:
```

Comment:
Is the double dash a typo?

Reply:
Intentional because inherit-and-override'are one level, levels being separated by `-`